locust/test/test_runners.py

```diff
@@ -159,6 +159,35 @@ class MyTestLocust(Locust):
             s = master.stats.get("/", "GET")
             self.assertEqual(700, s.median_response_time)
 
+    def test_slave_stats_report_with_none_response_times(self):
```

Comment:
Nice with tests that tests the whole cycle with master/slave communication!

It'd be nice to also have much simpler test in test_stats.py that just tests the None response time functionality for `StatsEntry`.